src/cf.data.pre

```diff
@@ -3284,9 +3289,14 @@ IFDEF: USE_SSL_CRTD
 DEFAULT: 32 startup=5 idle=1
 LOC: Ssl::TheConfig.ssl_crtdChildren
 DOC_START
-	The maximum number of processes spawn to service ssl server.
-	The maximum this may be safely set to is 32.
-	
+	Specifies the maximum number of certificate generation processes that
+	Squid may spawn (numberofchildren) and several related options. Using
+	too few helper processes queues requests. Using too many helpers
```

Comment:
It should be called either "helper processes" or just "helpers".

Reply:
Please see if you like the rephrased version better.